content/02.main-text.md

```diff
@@ -151,6 +151,18 @@ For example, figures can be specified using versioned URLs that refer to the cod
 In addition, manuscripts can be templated, so that numerical values or tables get inserted directly from the repository that created them.
 An [example repository](https://github.com/greenelab/manubot-rootstock) demonstrates Manubot's features and serves as a template for users to write their own manuscript with Manubot.
 
+Since its creation to facilitate the Deep Review, Manubot has been used to write a variety of scholarly documents.
+The [Sci-Hub Coverage Study](https://github.com/greenelab/scihub-manuscript) --- performed openly on GitHub from its inception --- investigated Sci-Hub's repository of pirated articles [@doi:10.7554/eLife.32822].
+Sci-Hub [reviewed](https://github.com/greenelab/scihub-manuscript/issues/17) the initial preprint from this study in a series of tweets, pointing out a major error in one of the analyses.
+Within hours, the authors used markdown's strikethrough formatting in Manubot to cross-out the errant sentences ([commit](https://github.com/greenelab/scihub-manuscript/commit/8fcd0cd665f6fb5f39bed7e26b940aa27d4770ba), [versioned manuscript](https://greenelab.github.io/scihub-manuscript/v/8fcd0cd665f6fb5f39bed7e26b940aa27d4770ba/)), thereby alerting readers to the mistake and preventing further propagation of misinformation.
+A larger [set of revisions](https://github.com/greenelab/scihub-manuscript/pull/19) that was merged a month later explained the error in more detail, and was included in a second version of the preprint.
+As such, continuous publishing via Manubot helped the authors address the error without delay, while retaining a clear version history of the process.
+Of interest, in 2017, the Sci-Hub Coverage Study preprint was the [most viewed](http://web.archive.org/web/20171221221858/http://www.prepubmed.org/top_preprints/) _PeerJ Preprint_, while the Deep Review was the most viewed _bioRxiv_ preprint [@doi:10.1038/d41586-017-08493-x].
```

Comment:
I do think that the collaborative writing contributed to the popularity of these papers, but we can't say how much of it was due to Manubot.  These are hot topics that would have gathered a lot of interest even if written in a traditional manner.

Should we save these two sentences about the popular preprints for the cover letter?

Reply:
I think it belongs in the manuscript since I think it will be impressive to many readers. There are two difficult to separate possibilities:

1. authoring via manuscript leads to impactful work
2. impactful researchers use manubot

Both possibilities are selling points for the tool. I don't think we overly conclude that the popularity of the preprints was possibility 1 and not 2.

> Should we save these two sentences about the popular preprints for the cover letter?

We can also mention in cover letter, but briefly.

Reply:
I see your point.  I'm okay leaving it in.

The linked "Top 200 PeerJ Preprints" shows Sci-Hub as the number 2 most viewed, and the 2017 year in review states deep review was the most viewed in the second quarter.  Do we have evidence about bioRxiv views for the entire year?

Reply:
See https://github.com/greenelab/manufund-2018/pull/2#discussion_r158384720:

> ![biorxiv-popularity](https://user-images.githubusercontent.com/1117703/42457640-f8764c08-8365-11e8-9670-2dc8e5512950.png)

@cgreene private messaged Richard Sever who commented on 2017-12-22 that the quarterly preprints in the Nature News article were:

> the most viewed preprints in 2017 by the quarter they were uploaded at the time the database query was performed

Putting Richard's documentation together with the view counts from 2017-12-21, I think it's fair to say it's the most viewed. Obviously, preprints from earlier in 2017 have an advantage based on this metric. I don't think we've ruled out preprints from before 2016 having more views in 2017 than Deep Review, but I think that's unlikely?

> The linked "Top 200 PeerJ Preprints" shows Sci-Hub as the number 2 most viewed

The most-viewed preprint in this list is from 2016. Deep Review is the top preprint from 2017

Reply:
Is it the word "yielded" that is too suggestive of causation?

Reply:
Hm, maybe that is it. But I've re-read the paragraph several times now and I think I'm okay with it as is. As you say above, we'll never know what is responsible for the popularity, so I don't think there is any more precise way of saying what you've said.

Reply:
Thanks for the additional support for the 2017 views.  The phrasing doesn't indicate to me that we only consider manuscripts written in 2017 though.  What about
> Of interest, the Sci-Hub Coverage Study preprint was the [most viewed](http://web.archive.org/web/20171221221858/http://www.prepubmed.org/top_preprints/) 2017 _PeerJ Preprint_, while the Deep Review was the most viewed 2017 _bioRxiv_ preprint [@doi:10.1038/d41586-017-08493-x].

It's a weaker writing style, but switching to the passive voice in the next line could help reduce any causal interpretations
> Hence, in the Manubot's first year, two of the most popular preprints were written using its collaborative, open, and review-driven authoring process.

Also in line 161, `Manubot's` instead of `the Manubot's` seems more consistent with our other uses.